mkdocs.yml

```diff
@@ -14,6 +14,8 @@ markdown_extensions:
     - admonition:
     - markdown_include.include:
         headingOffset: 1
+    - toc:
+        permalink: true
```

Comment:
What does this do, for my education?

Reply:
Deep linking of headers, to make them clickable and able to be shared as permalinks. When you mouseover them you'll see the "link" icon. Clicking it takes you to it, or right-clicking it allows you to copy it.

e.g. http://oobliss.com:8080/configuration/optional-settings/#nautobot_root

Without this enabled those anchors are still there, but it just doesn't have the UI mouseover feature to make it obvious.